python/dgl/distributed/standalone_kvstore.py

```diff
@@ -102,3 +102,10 @@ def count_nonzero(self, name):
             the number of nonzero in this data.
         """
         return F.count_nonzero(self._data[name])
+
+    def union(self, operand1_name, operand2_name, output_name):
```

Comment:
"""Compute the union of two mask arrays in the KVStore."""
    self._data[output_name][:] = ( 
        self._data[operand1_name] | self._data[operand2_name]
    )

Reply:
Define in as utils.py to avoid copypast?

Reply:
I don't think a simple or operator deserves a `utils.py` function.